storagenode/storagenodedb/infodb.go

```diff
@@ -217,6 +217,13 @@ func (db *InfoDB) Migration() *migrate.Migration {
 					)`,
 				},
 			},
+			{
+				Description: "Add creation date.",
+				Version:     6,
+				Action: migrate.SQL{
+					`ALTER TABLE pieceinfo ADD COLUMN piece_creation TIMESTAMP`,
```

Comment:
this probably should be not null

Reply:
maybe something like `ALTER TABLE pieceinfo ADD COLUMN piece_creation TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT 'epoch';`